src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -101,6 +101,22 @@ public function getAddress()
         return $this->hostname.':'.$this->port;
     }
 
+    /**
+     * @return string contains resolved hostname if available, empty string otherwise
+     */
+    public function getResolvedAddress()
```

Comment:
I'd suggest calling it getDisplayAddress actually (nothing is "resolved" to me)

Reply:
Do you recommend to reword the message displayed to users too? Currently it says `... (resolved as ...)`  I think it's easy to understand that way ... but may there's a better wording. Thanks!